src/routes/compilations.js

```diff
@@ -6,6 +6,28 @@ const adjustAudio = require("../middlewares/adjustAudio");
 const extractFrames = require("../middlewares/extractFrames");
 const findEditPoints = require("../middlewares/findEditPoints");
 
-router.post("/", adjustAudio, extractFrames, findEditPoints);
+const progressStatus = require("./progressStatus");
+
+router.post(
+  "/",
+  (req, res, next) => {
+    progressStatus.stage = "start";
+
+    next();
+  },
+  adjustAudio,
+  extractFrames,
+  findEditPoints,
+  // TODO. 아래 내용에 컨트롤러가 위치해야합니다.
+  // 컨트롤러 연결시 progressStatus.stage = "completed";가 들어가야 합니다.
+  /* (req, res, next) => {
+    progressStatus.stage = "completed";
+    next();
+  } */
+);
+
+router.get("/", (req, res) => {
+  res.json(progressStatus.stage);
+});
```

Comment:
민지님! get 요청에 status 코드 200과 함께 보내준다면 조금 더 명시적인 응답이 될 것 같습니다!
```suggestion
router.get("/", (req, res) => {
  res.status(200).json(progressStatus.stage);
});
```

Reply:
확인했습니다 민석님! 200으로 수정했습니다 👍